react/modules/TaskQueue.ts

```diff
@@ -9,6 +9,11 @@ export const TASK_CANCELLED_MSG =
 
 export type QueueEvent = 'Pending' | 'Fulfilled'
 
+export const STATUS: Record<string, QueueEvent> = {
+  PENDING: 'Pending',
+  FULFILLED: 'Fulfilled',
+}
```

Comment:
⛏ Consider moving it to a constants file.

Reply:
This was exported only to be used in the tests, so I don't see much value in moving this to a separate file for now.

Reply:
Ok, I lied, these values are used both by `OrderQueue` and the `TaskQueue` implementation itself. I transformed this into an enum, moved to a constants file and exported it with `OrderQueue`, so it can be used by the consuming apps.